scripts/make-search.js

```diff
@@ -80,7 +80,15 @@ console.log('Generating search index for documentation');
         relativePath.startsWith('assets/') ||
         relativePath.startsWith('dist/') ||
         filename === '_sidebar.md' ||
-        filename === '404.md'
+        filename === '404.md' ||
+        filename === 'border-radius.md' ||
```

Comment:
What's the rationale for excluding all these files below from the search?

Reply:
The rationale for hiding the border radius, color, elevation, spacing, and typography files is because I created new files that show our tokens & content (they're the ones linked in the sidebar under Styles), but you could still get to the orig Shoelace files through search & the information is actually not correct because it just shows the SL defaults. So I thought it'd be better to hide these from search so that people don't get to those page by accident & then mistakenly reference those values. The other hidden token files aren't really documented or used in Figma or our designs right now, so they're not technically incorrect but just not relevant yet, and since we can still get the same content from the Shoelace site (or even our own github repo, since the files are still there), I thought it'd be better to just hide them.